src/main/java/chess/domain/position/Row.java

```diff
@@ -0,0 +1,55 @@
+package chess.domain.position;
+
+import java.util.Map;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toMap;
+
+public class Row {
+    private final int value;
+    private static final Map<Integer, Row> CACHE = IntStream.rangeClosed(0, 7)
+            .boxed()
+            .collect(toMap(
+                    i -> i,
+                    Row::new
+            ));
```

Comment:
```suggestion
            .collect(toMap(i -> i, Row::new));
```
한칸을 내리는 것은 생각보다 많은 가독성을 요구해요. 라인이 너무 길어져서 내려야할 상황이 아니라면 과연 이게 잘 읽힐 수 있느냐를 항상 고민해보세요 :)